youtube_dl/extractor/nova.py

```diff
@@ -100,6 +104,21 @@ class NovaIE(InfoExtractor):
             'description': 'md5:f0a42dd239c26f61c28f19e62d20ef53',
             'thumbnail': r're:^https?://.*\.(?:jpg)',
         }
+    }, {
+        'url': 'https://novaplus.nova.cz/porad/televizni-noviny/epizoda/41099-televizni-noviny-8-1-2020',
+        'info_dict': {
+            'id': 'LWVmgbBh2tR',
+            'ext': 'mp4',
+            'title': '2020-01-08 Televizní noviny',
+            # 'description': 're:.*Sportovní noviny, Počasí Pořad je opatřen audiodeskripcí.*',
+            'thumbnail': r're:https?://.*\.jpg(\?.*)?',
+            # 'upload_date': '20200108'
+        },
+        'params': {
+            # rtmp download
+            'skip_download': True,
+        },
+        'add_ie': [NovaEmbedIE.ie_key()],
```

Comment:
I'm not sure if I need this

Reply:
Only if extraction goes through delegation to `NovaEmbedIE`.

Reply:
thanks